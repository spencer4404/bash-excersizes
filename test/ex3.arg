test/ex3.arg
