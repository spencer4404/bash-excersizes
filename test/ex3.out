The file "test/ex3.arg" exists.
The file "test/blahblahblah" does not exist.
